From 9ba3ace9095ffc7444c2e797986dc82b8d7dfae5 Mon Sep 17 00:00:00 2001
From: Jianwei Hu <jianwei.hu@windriver.com>
Date: Mon, 31 Jul 2017 10:59:03 +0800
Subject: [PATCH] deleted one readme file

---
 readme | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 readme

diff --git a/readme b/readme
deleted file mode 100644
index 925eccd..0000000
--- a/readme
+++ /dev/null
@@ -1 +0,0 @@
-hu
-- 
1.8.3.1

